.vscode/launch.json

```diff
@@ -16,33 +16,25 @@
 			"stopOnEntry": false,
 			"sourceMaps": true,
 			"outFiles": [
-				"${workspaceFolder}/out/**/*.js"
+				"${workspaceFolder}/../dist/**/*.js"
 			],
-			"preLaunchTask": "npm"
+			"preLaunchTask": "npm: webpack"
```

Comment:
As discussed offline, I failed to F5 because it failed to find the out/test stuff. We either need to remove dependency on test stuff when doing "Launch extension" or just do `npm: pretest` here like you have for "Launch Tests"

Reply:
Fixed